src/main/java/com/thealgorithms/misc/Sparcity.java

```diff
@@ -11,7 +11,9 @@
  * @author Ojasva Jain
  */
 
-class Sparcity {
+final class Sparcity {
```

Comment:
Could you rename this class?
```suggestion
final class Sparsity {
```